appinventor/appengine/src/com/google/appinventor/client/wizards/FileUploadWizard.java

```diff
@@ -79,119 +97,108 @@ public FileUploadWizard(FolderNode folderNode, FileUploadedCallback fileUploaded
    * @param acceptableTypes a collection of acceptable types, or null.
    * @param fileUploadedCallback callback to be executed after upload
    */
-  public FileUploadWizard(final FolderNode folderNode,
-      final Collection<String> acceptableTypes,
-      final FileUploadedCallback fileUploadedCallback) {
-    super(MESSAGES.fileUploadWizardCaption(), true, false);
+  public FileUploadWizard(final FolderNode p_folderNode,
+      final Collection<String> p_acceptableTypes,
+      final FileUploadedCallback p_fileUploadedCallback) {
+    folderNode = p_folderNode;
+    acceptableTypes = p_acceptableTypes;
+    fileUploadedCallback = p_fileUploadedCallback;
```

Comment:
Note that you could have just left the parameter names the same and done `this.folderNode = folderNode`, etc.